concrete/src/Database/Driver/PDOMySqlConcrete5/Driver.php

```diff
@@ -8,7 +8,6 @@
 /**
  * PDO MySql driver.
  *
- * @since 2.0
```

Comment:
Should add `@since 5.7.0` ?

Reply:
Well, every class is `@since 5.7.0`: it's the first release of the new concrete5 era.
If we add `@since 5.7.0`, we should add it in any class that exists since 5.7.0...

Reply:
I see.